src/integrations/log.rs

```diff
@@ -182,7 +186,24 @@ impl log::Log for Logger {
     fn log(&self, record: &log::Record<'_>) {
         if self.options.create_issue_for_record(record) {
             Hub::with_active(|hub| {
-                hub.capture_event(event_from_record(record, self.options.attach_stacktraces))
+                let event = if let Some(custom_event_from_record) =
+                    &self.options.custom_event_from_record
+                {
+                    let mut event = custom_event_from_record(record);
+                    if self.options.attach_stacktraces && event.exception.is_empty() {
+                        event.exception = vec![Exception {
```

Comment:
would be nice to extract this into a common function thats shared with `event_from_record`.
With that exposed, there is also no need to special case this here, we can just rely on the custom callback to attach the stacktrace.